books/items.py

```diff
@@ -0,0 +1,12 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class BooksItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
```

Comment:
The `pass` statement is a placeholder indicating that the class currently has no fields. Once you define fields, you can remove this line.

Reply:
this generated automaticly, i don`t this that i need to change this code